api/tests/test_cloud_record_summary.py

```diff
@@ -18,8 +15,134 @@ class CloudRecordSummaryTest(TestCase):
     """Tests GET requests to the Cloud Sumamry Record endpoint."""
 
     def setUp(self):
-        """Disable logging.INFO from appearing in test output."""
-        logging.disable(logging.INFO)
+        """Prevent logging from appearing in test output."""
```

Comment:
As discussed, there may be a way to buffer log output so it's only used if a test fails.